src/api/store/message.py

```diff
@@ -38,16 +42,55 @@ def get_logo(id):
         if com:
             return com.logo.file.url if com.logo else None
         
-    
 
-
-def get_message_recipients(audience, audience_type):
-    if not is_null(audience):
+def get_message_recipients(audience, audience_type, community_ids,sub_audience_type):
+    
+    if is_null(audience): return None
+    if community_ids and not isinstance(community_ids, list):
+        community_ids = community_ids.split(",")
+
+    if audience_type == "COMMUNITY_CONTACTS":
+        communities = Community.objects.all()
+        if audience != ALL:
```

Comment:
do you want it to be case sensitive?
```suggestion
        if audience.lower() != ALL.lower():
```